pom.xml

```diff
@@ -104,6 +96,11 @@
       <groupId>io.jenkins.plugins</groupId>
       <artifactId>plugin-util-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-math3</artifactId>
```

Comment:
This was already a dependency via `ssj`, I think it was just unused previously, but now we use it for `SimpleRegression`.